pkg/azuredisk/nodeserver_v2.go

```diff
@@ -93,7 +94,27 @@ func (d *DriverV2) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolume
 	}
 	defer d.volumeLocks.Release(diskURI)
 
-	lunStr, ok := req.PublishContext[consts.LUN]
+	azVolumeAttachment, getErr := d.crdProvisioner.GetAzVolumeAttachment(ctx, diskURI, d.NodeID)
+	if getErr != nil {
+		return nil, status.Errorf(codes.Internal, "failed to get current attachment state for volume (%s) and node (%s): %v", diskURI, d.NodeID, getErr)
```

Comment:
Omit "state" since we're getting the object, e.g. "...current attachment for..."